.circleci/config.yml

```diff
@@ -1265,71 +922,70 @@ workflows:
+      - node-couchbase:
+          matrix:
+            parameters:
+              node-version: ["12"]
+      - node-cucumber: *matrix-supported-node-versions
+      - node-dns: *matrix-supported-node-versions
+      - node-elasticsearch: *matrix-supported-node-versions
+      - node-express: *matrix-supported-node-versions
+      - node-fastify: *matrix-supported-node-versions
+      - node-fs: *matrix-supported-node-versions
+      - node-generic-pool: *matrix-supported-node-versions
+      - node-google-cloud-pubsub: *matrix-supported-node-versions
+      - node-graphql: *matrix-supported-node-versions
+      - node-grpc: *matrix-supported-node-versions
+      - node-hapi: *matrix-supported-node-versions
+      - node-http: *matrix-supported-node-versions
+      - node-http2: *matrix-supported-node-versions
+      - node-jest: *matrix-supported-node-versions
+      - node-kafkajs: *matrix-supported-node-versions
+      - node-knex: *matrix-supported-node-versions
+      - node-koa: *matrix-supported-node-versions
       # - node-limitd-client
-      - node-memcached
-      - node-microgateway-core
-      - node-mocha
-      - node-mongodb-core
-      - node-mongoose
-      - node-mysql
-      - node-net
-      - node-next
-      - node-oracledb
-      - node-paperplane
-      - node-pino
-      - node-postgres
-      - node-promise-js
-      - node-promise
-      - node-q
-      - node-redis
-      - node-restify
-      - node-router
-      - node-tedious
-      - node-when
-      - node-winston
+      - node-memcached: *matrix-supported-node-versions
+      - node-microgateway-core: *matrix-supported-node-versions
+      - node-mocha: *matrix-supported-node-versions
+      - node-mongodb-core: *matrix-supported-node-versions
+      - node-mongoose: *matrix-supported-node-versions
+      - node-mysql: *matrix-supported-node-versions
+      - node-net: *matrix-supported-node-versions
+      - node-next: *matrix-supported-node-versions
+      - node-oracledb: *matrix-supported-node-versions
+      - node-paperplane: *matrix-supported-node-versions
+      - node-pino: *matrix-supported-node-versions
+      # The pg version range we support doesn't work beyond Node 12. TODO fix this.
+      - node-postgres:
+          matrix:
+            parameters:
+              node-version: ["12"]
+      - node-promise-js: *matrix-supported-node-versions
+      - node-promise: *matrix-supported-node-versions
+      - node-q: *matrix-supported-node-versions
+      - node-redis: *matrix-supported-node-versions
+      - node-restify: *matrix-supported-node-versions
+      - node-router: *matrix-supported-node-versions
+      - node-tedious: *matrix-supported-node-versions
+      - node-when: *matrix-supported-node-versions
+      - node-winston: *matrix-supported-node-versions
       - codecov:
           requires:
-            - node-core-12
-            - node-core-14
-            - node-core-16
-            - node-core-latest
+            - node-core
```

Comment:
Does our current `codecov` usage work with matrixes?

Reply:
I don't know if everything is in there, but coverage hasn't gone *down* on this PR if I'm reading it right, so if we want we can give it some further effort in a future PR. How does that sound?